docs/source/get_started/kedro_concepts.md

```diff
@@ -63,20 +63,53 @@ The Kedro Data Catalog is the registry of all data sources that the project can
 
-The default Kedro project structure is as follows:
+Starting from Kedro 0.19, when you create a new project with `kedro new`, you can customise the structure by selecting which tools to include. Depending on your choices, the resulting structure may vary. Below, we outline the default project structure when all tools are selected and give an example with no tools selected.
+
+### Default Kedro project structure (all tools selected)
+
+If you select all tools during project creation, your project structure will look like this:
+
+```
+project-dir          # Parent directory of the template
+├── conf             # Project configuration files
+├── data             # Local project data (not committed to version control)
+├── docs             # Project documentation
+├── notebooks        # Project-related Jupyter notebooks (can be used for experimental code before moving the code to src)
+└── src              # Project source code
+├── tests            # Folder containing unit and integration tests
+├── .gitignore       # Hidden file that prevents staging of unnecessary files to `git`
+├── pyproject.toml   # Identifies the project root and contains configuration information
+├── README.md        # Project README
+├── requirements.txt # Project dependencies file
+```
+
+### Example Kedro project structure (no tools selected)
+
+If you select no tools, the resulting structure will be simpler:
 
 ```
-project-dir         # Parent directory of the template
-├── .gitignore      # Hidden file that prevents staging of unnecessary files to `git`
-├── conf            # Project configuration files
-├── data            # Local project data (not committed to version control)
-├── docs            # Project documentation
-├── notebooks       # Project-related Jupyter notebooks (can be used for experimental code before moving the code to src)
-├── pyproject.toml  # Identifies the project root and contains configuration information
-├── README.md       # Project README
-└── src             # Project source code
+project-dir          # Parent directory of the template
+├── conf             # Project configuration files
+├── notebooks        # Project-related Jupyter notebooks (can be used for experimental code before moving the code to src)
+└── src              # Project source code
+├── .gitignore       # Hidden file that prevents staging of unnecessary files to `git`
+├── pyproject.toml   # Identifies the project root and contains configuration information
+├── README.md        # Project README
+├── requirements.txt # Project dependencies file
 ```
 
+### Tool selection and resulting structure
```

Comment:
You can also add a link to the chapter where we discuss all the tools in more detail: https://docs.kedro.org/en/stable/starters/new_project_tools.html